rednotebook/data.py

```diff
@@ -199,13 +200,14 @@ def search(self, text, tags):
                     else:
                         add_text_to_results = True
             if add_text_to_results:
-                results.append(get_text_with_dots(self.text, 0, TEXT_RESULT_LENGTH))
+                results.append(get_text_with_dots(
+                    self.text, 0, TEXT_RESULT_LENGTH))
         elif text in str(self):
             # Date contains searched text.
-            results.append(get_text_with_dots(self.text, 0, TEXT_RESULT_LENGTH))
+            results.append(get_text_with_dots(
+                self.text, 0, TEXT_RESULT_LENGTH))
         else:
-            text_result = self.search_in_text(text)
-            if text_result:
+            if text_result := self.search_in_text(text):
```

Comment:
We'll need to bump the minimum Python version to 3.8 for the walrus operator, but I think that's fine.